### ISSUE TITLE
Captcha Detection?

### ISSUE THREAD

**Issue by pannatatm** [Frizz925/gbf-autopilot-core#16]
I'm currently using geasemonkey jquery script to detect captcha in the game. Most of time bot will stop itself when captcha popup come up because it cannot find button for next step. But in some case bot will try to refresh the page so if the code implement into bot so it can terminate itself.

function callDetect() {
    if ($('.prt-popup-header:contains(認証), .prt-popup-header:contains(Verif)').length) {
        var warning = new Audio("wani.mp3");
        warning.play();
    }
}

This is for old captcha only, I make it detect every 3 second outside of hash raid url. Work alert perfectly.

**Comment by Homusamu** [comment]
And also searching for div:contains(%keyword%) (like verification, tool, automated) for the new upcoming detect mechanic to make bot stop and play alert sound. This can be place only at support summon selection step. I'm pretty sure they gonna introduce new method of detect soon. Cuz the one that people got banned with, are very easy to fix. So we can assume that it meant for one time use only. since everyone got it at the same time and never happen again after that hour. (according to what I've read from some reddit post and japanese twitter)

**Comment by Frizz925** [comment]
Yeah it's really easy to fix but also clever enough to catch botters off guard. Old captcha methods still pop up during coops and poker session though.

**Comment by Homusamu** [comment]
@Frizz925 Here's full html part of original captcha. I got it during coop session. Was little panic so I forgot to copy the CSS and JS but this is probably enough for detection :p
```
<div class="pop-usual common-pop-error pop-show" data-token="" style="display: block; top: 144.833px;">
	<div class="prt-popup-header">Access Verification</div>
	<div class="prt-popup-body">
		<div class="txt-popup-body"><div class="">
	<div class="txt-message orange">
		Players who have continuously accessed the game<br>
		for long periods of time will sometimes<br>
		be required to verify access.<br>
	</div>
	<div class="txt-message">Enter the verification below.</div>
	<img class="image" src="c/i?t=??????????">

	<div class="prt-c-a-i-input">
		<div class="txt-c-a-i-message">1文字以上入力してください<br>読み取りづらい場合は一度送信してください</div>
		<div id="c-a-i-frm-group" class="frm-group">
			<textarea class="frm-message" cols="14" rows="1" wrap="soft" placeholder="Enter verification here."></textarea>
			<div class="btn-talk-message">Send</div>
		</div>
	</div>
</div></div>
	</div>
	<div class="prt-popup-footer">
	</div>
</div>
```
The ?????????? is some random number to captcha image. I removed it for safety reason as it might link to game server somehow.

**Comment by Frizz925** [comment]
Alright cool, thanks again
